homeassistant/components/lg_soundbar/__init__.py

```diff
@@ -1 +1,28 @@
 """The lg_soundbar component."""
+import logging
+
+from homeassistant import config_entries, core
+from homeassistant.const import Platform
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+PLATFORMS = [Platform.MEDIA_PLAYER]
+
+
+async def async_setup_entry(
+    hass: core.HomeAssistant, entry: config_entries.ConfigEntry
+) -> bool:
+    """Set up platform from a ConfigEntry."""
+    hass.data.setdefault(DOMAIN, {})
+    hass.config_entries.async_setup_platforms(entry, PLATFORMS)
```

Comment:
Verify the device is reachable before setting up the platforms, otherwise `raise ConfigEntryNotReady`